lib/cssLoadChecker/README.md

```diff
@@ -0,0 +1,19 @@
+# CSS Load Checker
+
+This module provides a means of checking if a CSS file has been loaded already or not. It does this by adding a test div to the page and checking the computed styles of it for the `display: none` style.
+
+```js
+var cssLoadChecker = require('bv-ui-core/lib/cssLoadChecker');
+
+if (cssLoadChecker.checkForCss('bv-verify-css-loaded')) {
```

Comment:
If it's not an async check, using a promise does feel like overengineering to me.